### ISSUE TITLE
Word highlight only works with Kaldi  

### ISSUE THREAD

**Issue by pietrop** [bbc/react-transcript-editor#37]
**TL;DR**: In the current implementation word highlight only works with Kaldi, and we have "lost" compatibility with importing other STT Json formats.

[See line `168` in  `TimedTextEditor`](https://github.com/bbc/react-transcript-editor/blob/master/src/lib/TranscriptEditor/TimedTextEditor/index.js#L168)
```js
const wordsArray = this.state.transcriptData.retval.words;
```

As explanation, 
- `this.state.transcriptData` can be different type of STT json.
- we then use the adapter module to convert to draftJs data model
- from then on we should reference draftJS data model to ensure compatibility with various formats 
- eg using`this.state.editorState` instead of `this.state.transcriptData`

Sudgestion is to re-write that function to pull out same data `this.state.editorState` content eg using [`getCurrentContent()`](https://draftjs.org/docs/api-reference-editor-state#getcurrentcontent).
Might need to be converted with [`convertToRaw`](https://draftjs.org/docs/api-reference-data-conversion#converttoraw) or there might be a better way to get the same data.
